### PR TITLE
upd(libdrm): `2.4.120` -> `2.4.122`

### PR DESCRIPTION
**Pull request by Xdavius**
fixed installdir to /usr/local do avoid conflicts with libdrm2 and all mesa stuffs...

### CODE REVIEW

**Comment by oklopfer** [comment]
Can you clarify why this *shouldn’t* be conflicting with those packages? It is my understanding that this package is equal to multiple different debs. I don’t think we should be changing to dir to /usr/local

**Review by oklopfer — CHANGES_REQUESTED** [review]
last thing was meant to be a review comment

**Comment by Xdavius** [comment]
> Can you clarify why this _shouldn’t_ be conflicting with those packages? It is my understanding that this package is equal to multiple different debs. I don’t think we should be changing to dir to /usr/local

because it breaks all the dependency cascade in /usr : 
https://packages.debian.org/fr/sid/libdrm-dev
https://deb.debian.org/debian/pool/main/libd/libdrm/libdrm_2.4.121-2.dsc

```Binary: libdrm-dev, libdrm2, libdrm-common, libdrm-tests, libdrm2-udeb, libdrm-intel1, libdrm-nouveau2, libdrm-radeon1, libdrm-omap1, libdrm-freedreno1, libdrm-exynos1, libdrm-tegra0, libdrm-amdgpu1, libdrm-etnaviv1```

If you can replace all of theses without breaking something, I need to learn how because I don't know how to solve this. 
If I can be sure that provides work, it's the solution.

**Comment by Xdavius** [comment]
In fact, I need to also provides I suppose to solve the cascade. and it should work in /usr also.

I update the provides but, tell me what you think about this.

**Comment by Xdavius** [comment]
```
The following packages will be REMOVED:
  fastfetch glffetch-git libdrm-amdgpu1 libdrm-intel1 libdrm-nouveau2 libdrm-radeon1 libdrm2 libegl-dev libegl-mesa0 libegl1 libegl1-mesa libgbm1
  libgl-dev libgl1 libgl1-mesa-dri libgl1-mesa-glx libglx-dev libglx-mesa0 libglx0 libosmesa6-dev mesa-vulkan-drivers
0 upgraded, 0 newly installed, 21 to remove and 0 not upgraded.
After this operation, 82.6 MB disk space will be freed.
(Reading database ... 55186 files and directories currently installed.)
Removing glffetch-git (1.0-pacstall1~git34ee3c6e) ...
Removing fastfetch (2.9.1-pacstall1~git5b76a0b4) ...
Do you want to remove the configuration files? [Y/n] 
Do you want to remove the left over cache? [Y/n] 
Removing mesa-vulkan-drivers:amd64 (22.3.6-1+deb12u1) ...
Removing libosmesa6-dev:amd64 (22.3.6-1+deb12u1) ...
Removing libegl-dev:amd64 (1.6.0-1) ...
Removing libegl1-mesa:amd64 (22.3.6-1+deb12u1) ...
Removing libegl1:amd64 (1.6.0-1) ...
Removing libegl-mesa0:amd64 (22.3.6-1+deb12u1) ...
Removing libgbm1:amd64 (22.3.6-1+deb12u1) ...
Removing libgl1-mesa-glx:amd64 (22.3.6-1+deb12u1) ...
Removing libgl-dev:amd64 (1.6.0-1) ...
Removing libgl1:amd64 (1.6.0-1) ...
Removing libglx-dev:amd64 (1.6.0-1) ...
Removing libglx0:amd64 (1.6.0-1) ...
Removing libglx-mesa0:amd64 (22.3.6-1+deb12u1) ...
Removing libgl1-mesa-dri:amd64 (22.3.6-1+deb12u1) ...
Removing libdrm-amdgpu1:amd64 (2.4.114-1+b1) ...
Removing libdrm-intel1:amd64 (2.4.114-1+b1) ...
Removing libdrm-nouveau2:amd64 (2.4.114-1+b1) ...
Removing libdrm-radeon1:amd64 (2.4.114-1+b1) ...
Removing libdrm2:amd64 (2.4.114-1+b1) ...
Processing triggers for libc-bin (2.36-9+deb12u7) ...
This script replaces libdrm-common. Do you want to proceed? [Y/n] 
```

**Comment by Xdavius** [comment]
with --prefix=/usr/local

```
The following NEW packages will be installed:
  libdrm-dev
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/423 kB of archives.
After this operation, 1348 kB of additional disk space will be used.
Get:1 /usr/src/pacstall/libdrm_2.4.122-pacstall1_amd64.deb libdrm-dev amd64 2.4.122-pacstall1 [423 kB]
Selecting previously unselected package libdrm-dev.
(Reading database ... 55766 files and directories currently installed.)
Preparing to unpack .../libdrm_2.4.122-pacstall1_amd64.deb ...
Unpacking libdrm-dev (2.4.122-pacstall1) ...
Setting up libdrm-dev (2.4.122-pacstall1) ...
Processing triggers for man-db (2.11.2-2) ...
[+] INFO: Performing post install operations
        [>] Storing pacscript
[+] INFO: Done installing libdrm
[+] INFO: Cleaning up
davius@distrobox-davius-debian ~/CLONES/pacstall-programs/packages/libdrm $ sudo apt list | grep libdrm2

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

libdrm2/stable,now 2.4.114-1+b1 amd64  [installé, automatique]
```

File: packages/libdrm/libdrm.pacscript [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 
 build() {
-  cd "${_archive}"
+  cd "drm-libdrm-${pkgver}"
   meson setup build \
-    -D udev=true \
-    -D etnaviv=disabled \
-    -D freedreno=disabled \
-    -D vc4=disabled \
-    -D valgrind=disabled \
```
Comment (oklopfer) [inline packages/libdrm/libdrm.pacscript:22]: Please re-disable these

**Review by oklopfer — CHANGES_REQUESTED** [review]

File: packages/libdrm/libdrm.pacscript [hunk 2]
```diff
@@ -1,30 +1,90 @@
...
-provides=("${pkgname}" "${pkgname}-dev")
-replaces=("${pkgname}-dev")
-source=("https://dri.freedesktop.org/${pkgname}/${pkgname}-${pkgver}.tar.xz")
-sha256sums=("3bf55363f76c7250946441ab51d3a6cc0ae518055c0ff017324ab76cdefb327a")
-maintainer=("Oren Klopfer <oren@taumoda.com>")
-
-build() {
-  cd "${_archive}"
+source=("https://gitlab.freedesktop.org/mesa/drm/-/archive/libdrm-${pkgver}/drm-libdrm-${pkgver}.tar.gz")
+arch=("amd64")
```
Comment (oklopfer) [inline packages/libdrm/libdrm.pacscript:23]: why was arm64 support removed? We need to keep it

**Review by oklopfer — CHANGES_REQUESTED** [review]

File: packages/libdrm/libdrm.pacscript [hunk 3]
```diff
@@ -1,30 +1,90 @@
...
+sha256sums=("7babfa22795dda27f32ed78b89bf93bfd341043e254bd42734cb569b602481d3")
+maintainer=("Oren Klopfer <oren@taumoda.com>" "xdavius <xdavius@github.com>")
+
+package_libdrm2() {
+  gives="libdrm2"
+  depends="${pkgbase}"
```
Comment (oklopfer) [inline packages/libdrm/libdrm.pacscript:29]: needs parentheses

**Review by oklopfer — CHANGES_REQUESTED** [review]

**Review by oklopfer — APPROVED** [review]

**Pull request merged**